cmd/main.go

```diff
@@ -0,0 +1,415 @@
+//
+// Copyright (c) 2019-2025 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package main
+
+import (
+	"flag"
+	"os"
+	"time"
+
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+
+	"github.com/eclipse-che/che-operator/controllers/namespacecache"
+	workspaceconfig "github.com/eclipse-che/che-operator/controllers/workspaceconfig"
+
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
+
+	dwr "github.com/devfile/devworkspace-operator/controllers/controller/devworkspacerouting"
+
+	"github.com/eclipse-che/che-operator/controllers/devworkspace/solver"
+	"github.com/eclipse-che/che-operator/controllers/usernamespace"
+
+	securityv1 "github.com/openshift/api/security/v1"
+
+	dwoApi "github.com/devfile/devworkspace-operator/apis/controller/v1alpha1"
+	"github.com/devfile/devworkspace-operator/pkg/infrastructure"
+	devworkspaceinfra "github.com/devfile/devworkspace-operator/pkg/infrastructure"
```

Comment:
nit, I see `IsOpenShift()` getting used in this class with these duplicated packages. Is it okay to delete one?

- `infrastructure.IsOpenShift()`
- `devworkspaceinfra.IsOpenShift()`